dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -87,8 +87,8 @@
   @eyes.force_full_page_screenshot = false
 
   @eyes.check_region(element, tag: identifier, match_timeout: MATCH_TIMEOUT, stitch_content: true)
-ensure
-  @eyes.force_full_page_screenshot = initial_force_full_page_screenshot if defined? initial_force_full_page_screenshot
+
+  @eyes.force_full_page_screenshot = initial_force_full_page_screenshot
```

Comment:
I ran into this, too, when I was trying to run these locally. seems like a sensible fix... not sure what the original intention was.